component/supportFunctions.go

```diff
@@ -0,0 +1,67 @@
+package component
+
+import (
+	p_buff "th2-grpc/th2_grpc_common"
+
+	"github.com/google/uuid"
+	timestamp "google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type Table struct {
+	Type    string        `json:"type"`
+	Rows    []interface{} `json:"rows"`
+	Headers []string      `json:"headers"`
+}
+
+func GetNewTable(headers ...string) *Table {
+	return &Table{
+		Type:    "table",
+		Rows:    nil,
+		Headers: headers,
+	}
+}
+func (table *Table) AddRow(args ...string) {
+	row := make(map[string]string)
+	for i, arg := range args {
+		row[table.Headers[i]] = arg
+	}
+	table.Rows = append(table.Rows, row)
+}
+
+func CreateEventID() *p_buff.EventID {
+	return &p_buff.EventID{Id: uuid.New().String()}
+}
+
+func GetTimestamp() *timestamp.Timestamp {
+	return timestamp.Now()
+}
```

Comment:
Looks like as redundant. Code will be more readable when we use timestamp.Now() explicitly